warning MSB3030: Could not copy the file "ApplicationInsights.config" because it was not found.

After upgrading to netcoreapp2.2 and upgrading Microsoft.ApplicationInsights.AspNetCore from 2.5 to 2.6.1 I'm getting this compilation warning:

```C:\Users\me\.nuget\packages\microsoft.applicationinsights.windowsserver\2.9.1\build\Microsoft.ApplicationInsights.WindowsServer.targets(9,5): warning MSB3030: Could not copy the file "ApplicationInsights.config" because it was not found.```

I'm using AI like this:

```
var host = new WebHostBuilder()
                ...Add lots of different stuff
                .UseStartup<Startup>()
                .UseApplicationInsights()
                .Build();
```

## Version Info 
SDK Version : 2.2.100
netcoreapp2.2
Windows 10

--- a/src/ServerTelemetryChannel/Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel.targets
+++ b/src/ServerTelemetryChannel/Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel.targets
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-    <Target Name="CopyApplicationInsightsConfigToOutputDirectory" AfterTargets="CopyFilesToOutputDirectory" >
+    <Target Name="CopyApplicationInsightsConfigToOutputDirectory" AfterTargets="CopyFilesToOutputDirectory" Condition="Exists('ApplicationInsights.config')" >
         <!--Currently we cannot set the config's "Copy to Output Directory" property to "PreserveNewest.
           Workaround is to use this targets file to manually copy into output path."-->
       

